src/parachains/zombienet.rs

```diff
@@ -515,3 +515,419 @@ impl Source {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
+
+	use super::*;
+	use anyhow::Result;
+
+	const CONFIG_FILE_PATH: &str = "./tests/zombienet.toml";
+	const TESTING_POLKADOT_VERSION: &str = "v1.7.0";
+	const POLKADOT_BINARY: &str = "polkadot-v1.7.0";
+	const POLKADOT_PARACHAIN_BINARY: &str = "polkadot-parachain-v1.7.0";
+
+	#[tokio::test]
+	async fn test_new_success() -> Result<()> {
```

Comment:
Are we running zombienet tests in our CI?
e.g. `cargo test --package pop-cli --bin pop -- parachains::zombienet::tests --nocapture `

Reply:
Is not running all tests in the CI? `cargo test --verbose`